utils/utils.py

```diff
@@ -369,3 +370,14 @@ def write_to_csv_by_chunks(file_to_write, output_file, write_mode="wb", chunksiz
                 buffer = f.read(chunksize)
         else:
             raise NotImplementedError(f"file' extension: [{file_to_write}] is not supported yet")
+
+
+def get_dir(dir_id):
+    directories = {"repository_root": repository_root_directory}
+
+    try:
+        directory = directories[dir_id]
+    except KeyError:
+        raise ValueError(f"{dir_id} is not defined")
+
+    return directories[dir_id]
```

Comment:
```suggestion
    try:
        return directories[dir_id]
    except KeyError:
        raise ValueError(f"{dir_id} is not known")
```

also the dictionary should probably be at module-level